dist/bash-completion/tpm2-tools/tss2_gettpm2object

```diff
@@ -0,0 +1,26 @@
+ bash completion for tss2_gettpm2object              -*- shell-script -*-
+
+_tss2_gettpm2object()
+{
+    local cur prev words cword split
+    _init_completion -s || return
+    case $prev in
+        -!(-*)h | --help)
+            COMPREPLY=( $(compgen -W "man no-man" -- "$cur") )
+            return;;
+        -!(-*)[o] | --tpm2object)
```

Comment:
How about using `-c | --key-context` or something similar to match the tpm2_* tools ?

Reply:
yes I agree, that would be better.